sagemaker-python-sdk/dgl_lifesci_bap/pytorch_bap.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>You could mention which objective metrics are available, and why you are using MAE here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Added this statement 'For this example, "mae" and "r2" is supported.' Just as a simple test, we are using MAE here.